core/consensus/babe/src/lib.rs

```diff
@@ -19,7 +19,6 @@
 //! BABE (Blind Assignment for Blockchain Extension) consensus in Substrate.
 
 #![forbid(unsafe_code, missing_docs, unused_must_use, unused_imports, unused_variables)]
```

Comment:
```suggestion
#![forbid(unsafe_code, missing_docs)]
```

Reply:
Originally I removed all forbid/deny but since Gav did the same in #3213 I reverted to avoid conflicts. The only thing I removed `forbid(unused_code)` was necessary to make this PR compile.